cmd/cmd.go

```diff
@@ -278,6 +278,18 @@ func generate(cmd *cobra.Command, model, prompt string) error {
 		}
 
 		if err := client.Generate(context.Background(), &request, fn); err != nil {
+			if strings.Contains(err.Error(), "failed to load model") {
+				// tell the user to check the server log, if it exists locally
+				home, nestedErr := os.UserHomeDir()
+				if nestedErr != nil {
+					// return the original error
+					return err
+				}
+				logPath := filepath.Join(home, ".ollama", "logs", "server.log")
```

Comment:
Is this the right path? Especially for a Mac app started ollama server?

Reply:
I believe so, this logs are set by the electron app like this:
`path.join(app.getPath('home'), '.ollama', 'logs', 'server.log')`

When the argument 'home' is used with app.getPath(), I believe the function will return the path of the user's home directory.
cc: @jmorganca to double check with his electron knowledge